db-commons/src/main/scala/org/bitcoins/db/AppConfig.scala

```diff
@@ -159,31 +159,110 @@ abstract class AppConfig extends StartStopAsync[Unit] with BitcoinSLogger {
     baseDatadir.resolve(lastDirname)
   }
 
+  lazy val driver: DatabaseDriver = {
```

Comment:
not neccesarily related to this PR, but we probably want to move all this database stuff into it's own trait? Not all things that subclasses of `AppConfig` have database components (like `KeyManagerAppConfig`). 

maybe something like 

```scala

trait DbAppConfig extends AppConfig { 
  lazy val driver: DatabaseDriver = {
      val driverName =
        getConfigString(s"bitcoin-s.$moduleName.db.driverName")
      DatabaseDriver.fromString(driverName)
    }

    //more methods down here...
}

```